cmd/ebpf/xdp/n3n6_entrypoint.c

```diff
@@ -95,7 +95,7 @@ static __always_inline enum xdp_action handle_n6_packet_ipv4(struct packet_conte
     if (qer->dl_gate_status != GATE_STATUS_OPEN)
         return XDP_DROP;
 
-    if (XDP_DROP == limit_rate_sliding_window(ctx->xdp_ctx, &qer->dl_start, qer->dl_maximum_bitrate))
+    if (XDP_DROP == limit_rate_sliding_window(ctx->xdp_ctx->data_end - ctx->xdp_ctx->data, &qer->dl_start, qer->dl_maximum_bitrate))
```

Comment:
It'll be better for readability to calculate packet size first, then use calculated size in function call.